view/theme/frio/templates/common_tabs.tpl

```diff
@@ -31,28 +33,33 @@
 			<ul class="tabs" role="menu">
 				{{foreach $tabs as $tab}}
 					{{if $tab.sel}}
-					<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"{{/if}}>{{$tab.label}}</a></li>
+						<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a
+								role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"
+								{{/if}}>{{$tab.label}}</a></li>
 					{{else}}
-					{{$exttabs[]=$tab}}
+						{{$exttabs[]=$tab}}
 					{{/if}}
 				{{/foreach}}
 			</ul>
 		</li>
 
 		{{* All others are moved to this dropdown menu *}}
-		<li class="pull-right">
+		<li>
 			<ul class="tabs tabs-extended">
 				<li class="dropdown">
-					<button type="button" class="btn-link dropdown-toggle" id="dropdownMenuTools-xs" data-toggle="dropdown" aria-expanded="false">
+					<button type="button" class="btn-link dropdown-toggle" id="dropdownMenuTools-xs"
+						data-toggle="dropdown" aria-expanded="false">
 						<i class="fa fa-chevron-down" aria-hidden="true"></i>
 					</button>
-					<ul class="dropdown-menu pull-right" role="menu" aria-labelledby="dropdownMenuTools">
+					<ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenuTools">
 						{{foreach $exttabs as $tab}}
-						<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"{{/if}}>{{$tab.label}}</a></li>
+							<li id="{{$tab.id}}-xs" role="presentation" {{if $tab.sel}} class="{{$tab.sel}}" {{/if}}><a
+									role="menuitem" href="{{$tab.url}}" {{if $tab.title}} title="{{$tab.title}}"
+									{{/if}}>{{$tab.label}}</a></li>
```

Comment:
```suggestion
							<li id="{{$tab.id}}-xs" role="presentation"{{if $tab.sel}} class="{{$tab.sel}}"{{/if}}>
								<a role="menuitem" href="{{$tab.url}}"{{if $tab.title}} title="{{$tab.title}}"{{/if}}>
									{{$tab.label}}
								</a>
							</li>
```

Reply:
No prob but not my fault! :joy:  It's quite time consuming to insert new line after each tags, currently all tpl files are jam-packed with tags and no room to breath. I believe my current line limit is ~120, I can try and make it lower, but I have no idea if vscode will apply this to smarty files.

Thanks

Reply:
I mean, you break it you fix it 😁

Reply:
Honestly this is so frustrating that sometimes I open a tpl file and abandon what I was going to do as soon as I see all the tags cramped up together lol